shakedown/dcos/service.py

```diff
@@ -225,6 +252,28 @@ def wait_for_mesos_task_removal(task_name, timeout_sec=120):
     return time_wait(lambda: mesos_task_not_present_predicate(task_name), timeout_seconds=timeout_sec)
 
 
+def delete_persistent_data(role, principal, zk_node):
```

Comment:
this same code has been attempted to be pushed into this repo several times... including once by me :)
this is to be handled without call janitor or docker and I believe is already handled by:  https://github.com/dcos/shakedown/blob/master/shakedown/dcos/zookeeper.py
at least for the zk part.